### PR TITLE
Getuser api

### PR DESCRIPTION
**Pull request by caroline-nejedlo**

### CODE REVIEW

File: backend/src/routes/account.ts [hunk 1]
```diff
@@ -284,16 +283,36 @@ router.post("/account-edit", async function (req, res) {
...
+      .json({
+        user,
+        message: "success",
+      });
+  } catch (error) {
+    return res.status(500).json({
+      message: error.message,
+    });
+  }
+});
```
Comment (pstoebenau) [inline backend/src/routes/account.ts]: Why is this here?

Comment (pstoebenau) [reply backend/src/routes/account.ts]: just remove your changes from this file.

**Review by pstoebenau — CHANGES_REQUESTED** [review]

File: backend/src/routes/account.ts (hunk 1 above)
Comment (caroline-nejedlo) [reply backend/src/routes/account.ts]: I removed it from account

**Review by pstoebenau — APPROVED** [review]
